android/java/org/chromium/chrome/browser/vpn/BraveVpnPlansActivity.java

```diff
@@ -0,0 +1,219 @@
+/**
+ * Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.chromium.chrome.browser.vpn;
+
```

Comment:
These are not used:
```
import static com.android.billingclient.api.BillingClient.SkuType.SUBS;
import android.view.ViewGroup;
import androidx.annotation.NonNull;
import com.android.billingclient.api.BillingClient;
import com.android.billingclient.api.BillingClientStateListener;
import com.android.billingclient.api.BillingFlowParams;
import com.android.billingclient.api.BillingResult;
import com.android.billingclient.api.Purchase;
import com.android.billingclient.api.PurchasesUpdatedListener;
import com.android.billingclient.api.SkuDetailsParams;
import com.android.billingclient.api.SkuDetailsResponseListener;
import org.json.JSONException;
import org.json.JSONObject;
import org.chromium.chrome.browser.init.AsyncInitializationActivity;
import org.chromium.ui.widget.Toast;
import java.text.DateFormat;
import java.util.ArrayList;
import java.util.Date;
import java.util.List;
```
Discussed with @deeppandya - we would like to make lint detect such things. Follow-up issue https://github.com/brave/brave-browser/issues/18095